FiiPrezent/Controllers/EventsController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using FiiPrezent.Entities;
+using FiiPrezent.Interfaces;
+using FiiPrezent.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace FiiPrezent.Controllers
+{
+    public class EventsController : Controller
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public EventsController(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task<IActionResult> Index()
+        {
+            return View(await _unitOfWork.Events.ListAllAsync());
+        }
+
+        public IActionResult Create()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Create(CreateEventViewModel model)
+        {
+            if (!ModelState.IsValid) return View(model);
+
+            // TODO: Add auto mapper
+            var @event = new Event
+            {
+                Id = new Guid(),
+                Name = model.Name,
+                Description = model.Description,
+                SecretCode = model.SecretCode
+            };
+
+            await _unitOfWork.Events.AddAsync(@event);
+            await _unitOfWork.CompletedAsync();
+
```

Comment:
Unit of Work nu e o ideea rea dar e mai important sa separam accesul la baza de date de controllere si view-uri...